samples/EdgeConfig/resources/edge/api/forecastweatherapi/resourceFiles/jsc/test.js

```diff
@@ -0,0 +1,3 @@
+//Trying within API
+var reqVerb = context.getVariable("request.verb");
```

Comment:
Can we try a non-debug statement here. Something like this,
https://github.com/apigee/api-platform-samples/blob/master/doc-samples/javascript-cookbook/apiproxy/resources/jsc/setHeaders.js

Reply:
Fixed it